includes/admin/downloads/metabox.php

```diff
@@ -919,33 +918,49 @@ function edd_render_refund_row( $post_id ) {
 	$refundability     = isset( $types[ $global_ability ] ) ? $types[ $global_ability ] : __( 'Unknown', 'easy-digital-downloads' );
 	$global_window     = edd_get_option( 'refund_window', 30 );
-	$edd_refundability = edd_get_download_refundability( $post_id );
 	$edd_refund_window = edd_get_download_refund_window( $post_id ); ?>
 
 	<div class="edd-product-options-wrapper">
-		<p><strong><?php _e( 'Refundability', 'easy-digital-downloads' ); ?><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>Refundability</strong>: Allow or disallow refunds for this specific product. When allowed, the refund window will be used on all future purchases.', 'easy-digital-downloads' ); ?>"></span></strong></p>
-		<p><?php echo EDD()->html->select( array(
-			'name'     => '_edd_refundability',
-			'options'  => $types,
-			'selected' => $edd_refundability,
-			'chosen'   => true,
-
-			'show_option_all'  => false,
-			'show_option_none' => '&mdash; ' . __( 'Use default', 'easy-digital-downloads' ) . ' &mdash;'
-		) ); ?></p>
 		<p>
-			<?php printf( __( 'Overrides default: %s', 'easy-digital-downloads' ), $refundability ); ?>
+			<strong>
+				<?php esc_html_e( 'Refunds', 'easy-digital-downloads' ); ?>
+				<span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php echo wp_kses( __( '<strong>Refundable</strong>: Allow or disallow refunds for this specific product. When allowed, the refund window will be used on all future purchases.<br /><strong>Refund Window</strong>: Limit the number of days this product can be refunded after purchasing.', 'easy-digital-downloads' ), array( 'strong' => true, 'br' => true ) ); ?>"></span>
+			</strong>
 		</p>
-	</div>
-	<div class="edd-product-options-wrapper">
-		<p><strong><?php _e( 'Refund Window', 'easy-digital-downloads' ); ?><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>Refund Window</strong>: Limit the number of days this product can be refunded after purchasing.', 'easy-digital-downloads' ); ?>"></span></strong></p>
+
 		<p>
-			<label>
-				<input class="small-text" name="_edd_refund_window" type="number" min="0" max="3650" step="1" value="<?php echo esc_attr( $edd_refund_window ); ?>" placeholder="<?php echo absint( $global_window ); ?>" />
-				<?php _e( 'Days', 'easy-digital-downloads' ); ?>
+			<label for="_edd_refundability" class="label--block">
+				<?php esc_html_e( 'Refund Status', 'easy-digital-downloads' ); ?>
 			</label>
+			<?php echo EDD()->html->select( array(
+				'name'             => '_edd_refundability',
+				'options'          => array_merge(
+					// Manually define a "none" option to set a blank value, vs. -1.
+					array(
+						'' => sprintf(
+							/** translators: Default refund status */
+							esc_html_x( 'Default (%1$s)', 'Download refund status', 'easy-digital-downloads' ),
+							ucwords( $refundability )
+						),
+					),
+					$types,
```

Comment:
```suggestion
					$types
```

Reply:
This needs to be removed as it causes a fatal error since this `$types` is a 2nd item in the `array_merge()`, instead of an array item.